android/src/main/java/com/amplitude/android/Configuration.kt

```diff
@@ -39,9 +39,10 @@ open class Configuration @JvmOverloads constructor(
     var locationListening: Boolean = true,
     var flushEventsOnClose: Boolean = true,
     var minTimeBetweenSessionsMillis: Long = MIN_TIME_BETWEEN_SESSIONS_MILLIS,
-    @Deprecated("Please use 'defaultTracking.sessions' instead.", ReplaceWith("defaultTracking.sessions"))
-    var trackingSessionEvents: Boolean = true,
-    var defaultTracking: DefaultTrackingOptions = DefaultTrackingOptions(),
+    trackingSessionEvents: Boolean = true,
+    @Suppress("DEPRECATION")
+    defaultTracking: DefaultTrackingOptions = DefaultTrackingOptions(),
+    val autocapture: MutableSet<AutocaptureOption> = mutableSetOf(AutocaptureOption.SESSIONS),
```

Comment:
As we discussed yesterday, shouldn't we use keep `autocapture` immutable as plugin installation is only done once?

Reply:
b4c09984d035a3b408719a2e06c3d42eb4c2c01a should make this mutable.